src/main/java/org/jenkins/tools/test/PluginCompatTester.java

```diff
@@ -388,20 +390,20 @@ && new VersionNumber(coreCoordinates.version).compareTo(new VersionNumber("1.485
         }
 
         if (failed && config.isFailOnError()) {
-            throw new AbortException("Execution was aborted due to the failure in a plugin test (-failOnError is set)");
+            throw new RuntimeException("Execution was aborted due to the failure in a plugin test (-failOnError is set)");
```

Comment:
This type was coming from Jenkins core. Nothing specifically catches it, so a generic `RuntimeException` is functionally equivalent.